models/maisi_ct_generative/configs/metadata.json

```diff
@@ -1,7 +1,8 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_generator_ldm_20240318.json",
-    "version": "0.4.2",
+    "version": "0.4.3",
     "changelog": {
+        "0.4.3": "remove download large files",
```

Comment:
Please also add texts like "adding weights_only=False in the model weight loading" in this line. Thanks.

Reply:
Hi @dongyang0122, I update this part to "add weights_only when loading weights" since I didn't set all of them to False, only two of them include other parameter info so I set to False.